Rakefile

```diff
@@ -1,8 +1,8 @@
 require 'tmpdir'
 
-desc 'Run unit tests on iOS 9.3 and 10.3'
+desc 'Run unit tests on iOS 11.0'
 task :test do
-  sh "set -o pipefail && xcodebuild -workspace 'Bootstrap/Bootstrap.xcworkspace' -sdk 'iphonesimulator' -scheme 'Bootstrap' -destination 'name=iPhone 6,OS=10.3.1' clean build test | xcpretty --color --simple"
```

Comment:
xcpretty was obscuring details about test failures, so I disabled it. I should add it back in.